network/p2p/validators.go

```diff
@@ -0,0 +1,76 @@
+// Copyright (C) 2019-2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package p2p
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/snow/validators"
+	"github.com/ava-labs/avalanchego/utils/set"
+)
+
+const maxValidatorSetStaleness = time.Minute
```

Comment:
Should we make this configurable?

Reply:
Originally didn't make this configurable because I wanted to remove this whole refresh logic and not cause a breaking change. My concern was probably premature since I don't think anyone is using SDK at the moment, so I will make this configurable.